src/components/NavigationSection.svelte

```diff
@@ -74,26 +73,26 @@
       return {
         busy: false,
         closed: true,
-        valve: false
+        isFetching: true
```

Comment:
@m4dz I didn't get any problem in my tests but it's using the dev environment, let's see in the prod, what do you think? Do you still have the issue with the setting panel?